pkg/telemetry/marketing/telemeter.go

```diff
@@ -0,0 +1,28 @@
+package marketing
+
+import (
+	"github.com/stackrox/rox/pkg/env"
+)
+
+// Telemeter defines a common interface for telemetry gatherers.
+type Telemeter interface {
+	Start()
+	Stop()
+	Identify(props map[string]any)
+	Track(event string)
```

Comment:
How about having a type for `event`.

Reply:
That's basically an event name. Do you think we need `type Event string`?